build_tools/github_action/portable_release_note_script.py

```
@@ -0,0 +1,40 @@
+from configure_ci import set_github_output
+import json
+import os
+from urllib.request import urlopen, Request
+
+TAG_NAME = os.getenv("TAG_NAME")
+VERSION = os.getenv("VERSION")
+
+def run():
+    github_release_url = (
+        f"https://api.github.com/repos/ROCm/TheRock/releases/tags/{TAG_NAME}"
+    )
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    # If GITHUB_TOKEN environment variable is available, include it in the API request to avoid a lower rate limit
+    gh_token = os.getenv("GITHUB_TOKEN", "")
+    if gh_token:
+        headers["Authentication"] = f"Bearer {gh_token}"
+
+    request = Request(github_release_url, headers=headers)
+    with urlopen(request) as response:
+        if response.status == 403:
+            print(
+                f"Error when retrieving GitHub release assets for release tag '{TAG_NAME}'. This is most likely a rate limiting issue, so please try again"
+            )
+            return
```

Any reason not to raise an error instead of returning?